Assets/Scripts/Model/Player.cs

```diff
@@ -21,5 +30,45 @@ public void AddToScoreLong(long bonus)
         {
             score += bonus;
         }
+
+        protected override void Kill()
+        {
+            FirstPersonController controller = gameObject.GetComponent<FirstPersonController>();
+            controller.DisableMoving();
+            wait = true;
+            StartCoroutine(KillCoroutine());
+        }
+
+        protected override void EndGame()
+        {
+            FirstPersonController controller = gameObject.GetComponent<FirstPersonController>();
+            controller.DisableMoving();
+            wait = true;
+            StartCoroutine(EndGameCoroutine());
+        }
+
+        private IEnumerator KillCoroutine()
+        {
+            if (wait)
+            {
+                wait = false;
+                yield return new WaitForSeconds(1);
+            }
+                EditorUtility.DisplayDialog("Bomberman3D", "You are dead.", "Repeat level");
```

Comment:
Nie wiem czy wyświetlanie dialogów to najlepszy pomysł, bo coś u mnie one nie działają zbyt dobrze (może wina Linux'a). Na razie niech zostaną, ale w przyszłości można pomyśleć nad innym rozwiązaniem (jakiś tekst na scenie + np. kliknięcie spacji kontynuuje).